client/src/components/SiteFooter/SiteFooter.jsx

```diff
@@ -13,7 +13,8 @@ const SiteFooter = () => {
         <Col xs={12} lg className="my-auto">
           <p className="text-light">
             <small>
-              All FINAL FANTASY&reg; XI content and images &copy; 2002-2020{' '}
+              All FINAL FANTASY&reg; XI content and images &copy; 2002-
+              {new Date().getFullYear()}{' '}
               <a href="https://www.square-enix.com/" className="text-white">
                 SQUARE ENIX
```

Comment:
It may be best to include their full legal name here. In addition, I'm not sure if it's good idea to link to their website. The functionality is great though. Thanks!

> SQUARE ENIX CO., LTD. All Rights Reserved.

Reply:
Yes, having the link makes logical sense, but I think it gives them referrer tracking data.